client/services/Hooks.ts

```diff
@@ -99,11 +99,27 @@ function withAuthModal(isLoggedIn: boolean, cb: (...any) => any) {
   };
 }
 
+function useKeyDown(handler: (e: KeyboardEvent) => void, eventName: string) {
```

Comment:
`eventName` is not a good choice for this variable name. It makes me think of DOM events (eg: `onclick`, `onscroll`, etc). 

Let's rename to `code` and also add a comment here explaining this refers to the `KeyboardEvent.code` property.

Also, please reorder the variables, have `code` first.